dashboard/app/assets/javascripts/header.js

```diff
@@ -56,7 +53,7 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId, userI
   }
   var progressContainer = $('.progress_container');
   stageData.levels.forEach(function(level, index, levels) {
-    var status = (levelProgress[level.id] || {}).status || 'not_tried';
+    var status = mergeProgress((progressData.levels[level.id] || {}).result, clientLevelProgress[level.id]);
```

Comment:
Should be defended here: https://github.com/code-dot-org/code-dot-org/pull/5676/files#diff-b6f39a4caef53f05fd5c48d51ec6f41aR34